draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -193,6 +196,33 @@ MUST expose a different mDNS name for each address.
 
 ### Implementation Guidance
 
+#### Registration and Responses
+
+If the ICE agent skips both probing and announcement sending, the ICE agent
+does not send any mDNS message on the network. In that case, the ICE agent is
+expected to answer to any query matching the registered name.
+
+Sending an mDNS announcement will usually allow the ICE agent receiving the
+remote ICE candidate containing the mDNS name to already have the information
+useful to resolve the mDNS name if the mDNS ICE candidate can actually be used.
+If the mDNS announcement is not sent, this can increase the connection setup
+time.
+
+Sending the mDNS announcement to the network can be delayed, for instance due
+to rate limits. An ICE agent can provide the candidate to the web application
+before the mDNS announcement is sent on the network.
+
+When processing an mDNS query message that matches a registered mDNS name and
+the mDNS query message has its response-unicast bit set, the ICE agent SHOULD
+answer through unicast and SHOULD NOT send an mDNS announcement for the mDNS
+name. This will limit the multicast traffic as the announcement is probably not
+useful to any other node of the network.
+
+TODO: Verify the 'SHOULD NOT' rule interaction with
```

Comment:
I think the 'SHOULD NOT' rule above plays well with 7.3, and the latter is about consolidating queries with other hosts. I don't see any clause in 6762 about how to handle pending announcements if there is already a prior response, say because of rate limiting scheme.